### PR TITLE
test infra

### PR DESCRIPTION
**Pull request by jburel**
Use the new test infra

Make sure travis is green

### CODE REVIEW

File: .omeroci/app-config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+set -e
+set -u
+OMERO_VENV=${OMERO_VENT:-/opt/omero/web/venv}
```
Comment (manics) [inline .omeroci/app-config]: typo in `OMERO_VENT`

**Review by manics — CHANGES_REQUESTED** [review]
Looks fine part from a typo.

Shame there aren't any tests!

File: .omeroci/app-config (hunk 1 above)
Comment (jburel) [reply .omeroci/app-config]: Discussed today with @joshmoore we are going to remove that file by default and configure the app directly in test-infra

**Comment by jburel** [comment]
I will revisit this PR when https://github.com/openmicroscopy/omero-test-infra/pull/3 is merged and the modification mentioned above integrated

**Comment by joshmoore** [comment]
In terms of a simple activation, :+1: (Green)

**Pull request merged**